### ISSUE TITLE
[Bug]: Unable to fetch collection schedule for whittlesea_vic_gov_au (Website redesign. API has been changed.)

### ISSUE THREAD

**Issue by EthanBezz** [mampfes/hacs_waste_collection_schedule#2324]
### I Have A Problem With:

A specific source

### What's Your Problem

It seems that Whittlesea Council have just launched a major redesign of their website and, with it, changed the API for the bin collection schedule.

Links to compare old vs new website:
https://web.archive.org/web/20240722085426/https://www.whittlesea.vic.gov.au/
https://www.whittlesea.vic.gov.au/

Links to compare old vs new API: (Using the address of the Whittlesea Council building)
https://www.whittlesea.vic.gov.au/umbraco/api/vicmap/GetAddressResultsUsingArcGis/?address=25+Ferres+Boulevard+South+Morang+3752
https://www.whittlesea.vic.gov.au/ocapi/Public/myarea/wasteservices?geolocationid=2622da61-0678-4bf0-b551-3543b8780b9c&ocsvclang=en-AU&pageLink=/$b9015858-988c-48a4-9473-7c193df083e4$/My-Neighbourhood

### Source (if relevant)

whittlesea_vic_gov_au

### Logs

```Shell
fetch failed for source Whittlesea City Council: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 145, in fetch entries = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/whittlesea_vic_gov_au.py", line 63, in fetch r.raise_for_status() File "/usr/local/lib/python3.12/site-packages/requests/models.py", line 1024, in raise_for_status raise HTTPError(http_error_msg, response=self) requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://www.whittlesea.vic.gov.au/umbraco/api/vicmap/GetAddressResultsUsingArcGis/?address=25+Ferres+Boulevard+South+Morang+3752
```

### Checklist Source Error

- [ ] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by EthanBezz** [comment]
I've created pull request #2326 which updates whittlesea_vic_gov_au to use the OpenCities API, and also updates the documentation the reflect changes to the configuration, which are breaking changes.

**Comment by 5ila5** [comment]
fixed by #2326

This will be part of the next release or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
